tools/tests/test_osqueryd.py

```diff
@@ -43,20 +44,19 @@ def test_2_daemon_with_option(self):
 
         self.assertTrue(daemon.isAlive())
 
+        info_path = os.path.join(logger_path, "osqueryd.INFO*")
         def info_exists():
-            info_path = test_base.getLatestInfoLog(logger_path)
-            return os.path.exists(info_path)
+            return len(glob.glob(info_path)) > 0
 
         # Wait for the daemon to flush to GLOG.
         test_base.expectTrue(info_exists)
 
         # Assign the variable after we have assurances it exists
-        info_path = test_base.getLatestInfoLog(logger_path)
-        self.assertTrue(os.path.exists(info_path))
+        self.assertTrue(info_exists())
 
         # Lastly, verify that we have permission to read the file
         data = ''
-        with open(info_path, 'r') as fh:
+        with open(glob.glob(info_path)[0], 'r') as fh:
```

Comment:
Mind checking the `len` on this before accessing? Maybe print out a debug message or fail the test? Or do we prefer the stack trace?